api-extractor-output/customerio-reactnative.api.md

```diff
@@ -78,7 +78,9 @@ export class CustomerIO {
     // (undocumented)
     static readonly inAppMessaging: CustomerIOInAppMessaging;
     static readonly initialize: (config: CioConfig) => Promise<void>;
+    // @deprecated
     static readonly isInitialized: () => boolean;
+    static readonly isInitializedAsync: () => Promise<boolean>;

```

Comment:
Couldn’t think of a better name. Making `isInitialized` `async` would be a breaking change. I was thinking that when we do public API change, we could remove `isInitializedAsync` and expose `isInitialized` again as `async`. Still not ideal though, so open to suggestions.

Reply:
If we need to return any value from native SDK, it has to be `async` so the `promise` can resolve without blocking the UI.